Formula/mesa.rb

```diff
@@ -13,6 +13,7 @@ class Mesa < Formula
 
   option "without-test", "Skip compile-time tests"
   option "with-static", "Build static libraries (not recommended)"
+  option "with-only-cpu", "without graphics hardware, and OSMesa"
```

Comment:
rename this to `"without-gpu"`.
Capitalize  the first word in the explanation: `Build without GPU hardware`.

Does it worth splitting it into two options? `without-gpu` and `without-osmesa`?

Reply:
``without-gnu`` is OK. But ``without-osmesa`` is not my intention. Maybe ``--enable-gallium-osmesa`` would be better in favor of the newer "gallium" OSMesa.